src/referencePackages/src/microsoft.bcl.asyncinterfaces/5.0.0/Microsoft.Bcl.AsyncInterfaces.5.0.0.csproj

```diff
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net461</TargetFrameworks>
+    <NuspecFile>$(ArtifactsBinDir)microsoft.bcl.asyncinterfaces/5.0.0/microsoft.bcl.asyncinterfaces.nuspec</NuspecFile>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+  <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <OutputPath>$(ArtifactsBinDir)microsoft.bcl.asyncinterfaces/5.0.0/ref/</OutputPath>
+    <IntermediateOutputPath>$(ArtifactsObjDir)microsoft.bcl.asyncinterfaces/5.0.0</IntermediateOutputPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <OutputPath>$(ArtifactsBinDir)microsoft.bcl.asyncinterfaces/5.0.0/lib/</OutputPath>
```

Comment:
Are these generated?  `$(ArtifactsBinDir)microsoft.bcl.asyncinterfaces/5.0.0/` is duplicated 6 times.  Utilizing a common variable would the reduce chances of inconsistencies if the projects are intended to be hand edited..

Reply:
Yes.  They are generated.  Good suggestion.  I filed https://github.com/dotnet/source-build/issues/2449